tests/symbolicator/__init__.py

```diff
@@ -26,14 +27,43 @@ def strip_frame(frame):
     return frame
 
 
+NORMALIZED_REGISTERS = {}
+
+
+def normalize_register(name):
+    return NORMALIZED_REGISTERS.get(name, name)
+
+
 def strip_stacktrace(stacktrace):
     if stacktrace:
         stacktrace = dict(stacktrace)
         stacktrace["frames"] = [strip_frame(x) for x in stacktrace.get("frames") or ()]
+        try:
+            stacktrace["registers"] = {
+                normalize_register(k): v for k, v in stacktrace["registers"].items()
+            }
+        except KeyError:
+            pass
```

Comment:
Not really sure why it's worth keeping all this code if NORMALIZED_REGISTERS is empty anyway.  It's not like this is going to come back anytime soon is it?